lib/helpers/backend/makeFetchRequest.ts

```diff
@@ -0,0 +1,29 @@
+import { StatusCodes } from "http-status-codes";
+
+export async function makeFetchRequest(endpoint : string, returnType : string, options? : Object, milliInterval? : number, maxRetry? : number) : Promise<Object | string | null> {
```

Comment:
Consider accepting a generic type parameter for the response type instead (https://www.typescriptlang.org/docs/handbook/2/generics.html)

Reply:
Options type should be `RequestInit` for `fetch` (or `AxiosRequestConfig` for `axios`)

Reply:
Axios is being used instead in the latest commit. returnType was removed.